resources/definitions/fdmprinter.def.json

```diff
@@ -816,6 +816,15 @@
                     "minimum_value_warning": "0",
                     "maximum_value_warning": "machine_nozzle_size",
                     "settable_per_mesh": true
+                },"outer_inset_first": 
+                {
+                    "label": "Print Outer Inset First",
+                    "description": "Prints insets in order of outside to inside when selected. This can help improve dimensional accuracy in x and y when using a high viscosity plastic like ABS however it can decrease outer surface print quality, especially overhangs.",
+                    "type": "bool",
+                    "default_value": false,
+                    "settable_per_mesh": false,
```

Comment:
Actually you retrieve it from the mesh setting base `mesh->getSettingBoolean("outer_inset_first")` so this line must be true.

That also means that `"settable_per_extruder": true,` and `"settable_per_meshgroup": true`, but because that is the default for such settings, you may leave the below two out.